gensim/models/utils_any2vec_fast.pyx

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env cython
+# cython: boundscheck=False
+# cython: wraparound=False
+# cython: cdivision=True
+# coding: utf-8
+
+def ft_hash(unicode string):
+    cdef unsigned int h = 2166136261
+    for c in string:
+        h ^= ord(c)
+        h *= 16777619
+    return h
+
+
+cpdef compute_ngrams(word, unsigned int min_n, unsigned int max_n):
```

Comment:
why not `nogil`, you can fix type for `word`.

Reply:
Fixing the type for `word` doesn't really work, since the token might be a `str` on Python 2.7. `nogil` won't work either, since the function returns a Python object.